test/fixtures/rules/std_rule_cases/L016.yml

```diff
@@ -325,15 +330,129 @@ test_fail_templated_comment_clause:
   pass_str: |
     SELECT *
         {# ........................................................................... #}
     FROM table
   configs:
     rules:
       L016:
-        ignore_comment_clauses: true
+        ignore_comment_lines: true
     templater:
       jinja:
         context: {}
+
+test_fail_operator_precedence_1:
+  # Make sure we split at the + operator.
+  fail_str: |
+    select
+        ISNULL(count, '0') * 10000 + ISNULL(planned, 100)
+    from blah
+  fix_str: |
+    select
+        ISNULL(count, '0') * 10000
+        + ISNULL(planned, 100)
+    from blah
+  configs:
+    indentation:
+      max_line_length: 30
+
+test_fail_operator_precedence_2:
+  # Make sure we split at the AND operator.
+  fail_str: |
+    select
+        recommendation_list[ORDINAL(1)] = 'uses_small_subject_line' AND uses_small_subject_line != CAST(effect_size_list[ORDINAL(1)] AS FLOAT64)
+    from blah
+  fix_str: |
+    select
+        recommendation_list[ORDINAL(1)] = 'uses_small_subject_line'
+        AND uses_small_subject_line != CAST(effect_size_list[ORDINAL(1)] AS FLOAT64)
+    from blah
+  configs:
+    indentation:
+      max_line_length: 120
+
+test_fail_operator_precedence_3:
+  # Stretching cases for operators and comma
+  fail_str: |
+    select
+        a, b + c, long_name + long_name * long_name - long_name as foo, long_name AND long_name OR long_name OR long_name as bar
+    from blah
+  fix_str: |
+    select
+        a,
+        b + c,
+        long_name
+        + long_name * long_name
+        - long_name as foo,
+        long_name AND long_name
+        OR long_name
+        OR long_name as bar
+    from blah
+  configs:
+    indentation:
+      max_line_length: 30
+
+test_pass_long_multiline_jinja:
+  # None of the lines are longer than 30
+  # but the whole tag is. It shouldn't
+  # cause issues.
+  pass_str: |
+    select
+        {{
+          1 + 2 + 3 + 4 + 5
+          + 6 + 7 + 8 + 9 + 10
+        }}
+    from blah
+  configs:
+    indentation:
+      max_line_length: 30
+
+test_fail_long_inline_statement:
+  # Tests that breaks happen between clauses properly
+  fail_str: |
+    select distinct a + b from c join d using (e) where f = g and h = i order by j
+  fix_str: |
+    select distinct a + b
+    from c
+    join d using (e)
+    where f = g and h = i
+    order by j
+  configs:
+    indentation:
+      max_line_length: 50
+
+test_pass_check_off_1:
+  # Tests that we can disable the check
+  pass_str: |
+    select my_really_really_really_really_really_really_really_really_really_really_really_long_var from tbl
+  configs:
+    indentation:
+      max_line_length: 0
+
+test_pass_check_off_2:
+  # Tests that we can disable the check
+  pass_str: |
+    select my_really_really_really_really_really_really_really_really_really_really_really_long_var from tbl
+  configs:
+    indentation:
+      max_line_length: -1
+
+test_comment_move_mid_query:
+  # Tests that we can disable the check
```

Comment:
>disable the check

What do you mean here? Is code plus comment allowed to exceed the max line limit?